docs/associations/source/reference/level2_asn_technical.rst

```diff
@@ -0,0 +1,151 @@
+.. _asn-level2-techspecs:
+
+Level 2 Associations: Technical Specifications
+==============================================
+
+Logical Structure
+-----------------
+
+Independent of the actual format, all Level 3 associations have the
+following structure. Again, the structure is defined and enforced by
+the Level 3 schema
+
+  * Top level, or meta, key/values
+  * List of products, each consisting of
+    
+    * Output product name template
+    * List of exposure members, each consisting of
+      
+      * filename of the input exposure
+      * Type of exposure
+      * Errors from the observatory log
+      * Association Candidates this exposure belongs to
+
+.. _asn-level2-example:
+   
+Example Association
+-------------------
+
+The following example will be used to explain the contents of an association::
+
+    {
+        "asn_rule": "Asn_Lv2Image",
+        "asn_pool": "jw82600_001_20160304T145416_pool",
+        "program": "82600",
+        "asn_type": "image2",
```

Comment:
Not a really big deal, but since the example files reference "lrs", which are spectra, it might be more appropriate to label the asn_type of this example as "spec2", rather than "image2."